<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventas</class>
 <widget class="QWidget" name="Ventas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventas</string>
  </property>
  <widget class="QComboBox" name="FiltroSucursales">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>291</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CrearVenta">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EditarVenta">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>420</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QTableView" name="TablaVentas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>700</width>
     <height>320</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="EliminarVenta">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackToMenu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Busqueda">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>241</width>
     <height>30</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Ingrese lo que desea buscar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BuscarLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
